packages/create-fuels/src/cli.ts

```diff
@@ -113,6 +115,19 @@ function writeEnvFile(envFilePath: string, programsToInclude: ProgramsToInclude)
   writeFileSync(envFilePath, newFileContents);
 }
 
+async function promptForFuelUpInstall() {
+  const shouldInstallFuelUp = await prompts(
+    {
+      type: 'confirm',
+      name: 'shouldInstallFuelUp',
+      message: "You don't have `fuelup` installed. Would you like us to install it for you?",
```

Comment:
I think we should also include why we are installing `fuelup` since it will be needed.
```suggestion
      message: "It seems you don't have `fuelup` installed, it's needed to manage the fuel toolchain. Would you like us to install it for you?",
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2149/commits/4500048497ac9939819900843906eb84cf4ca0be